docker/puppet-agent-alpine/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apk add --no-cache cmake boost-dev make curl git curl-dev ruby ruby-dev gcc
 ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
 ENV PATH $PATH:$JAVA_HOME/jre/bin:$JAVA_HOME/bin
 
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
```

Comment:
alpine doesn't have /bin/bash

Reply:
No, ash is one of the names of the shell on alpine. I use this name because if you leave /bin/sh here hadolint can't tell that you have a shell that supports pipefail.

Reply:
👍